region/norfair/east.json

```diff
@@ -773,6 +773,13 @@
           "utility": [ "farming" ]
         }
       ],
+      "obstacles": [
```

Comment:
Why are `ID` and `Name` capitalized?

Reply:
Cause I messed up :P

Reply:
Is this obstacle for the future case of wanting to use the farming and then go back?

Reply:
Yes. Same reason I did it in the Kronic Boost Room, where the Violas are farmable for PBs.

By opposition, I didn't do it in Ice Beam Gate Room to get to the small Dessgeegas (who are also farmable for PBs) because I figured it's only a difference of one PB. But thinking about it again, maybe it was worth doing?

Reply:
Nah